README.md

```diff
@@ -53,242 +22,248 @@ Human Essentials is one of many projects initiated and run by Ruby for Good. You
-```
+and run the spec using this command: `MAGIC_TEST=1 NOT_HEADLESS=true bundle exec rspec <path_to_spec>`
 
 **See videos of it in action [here](https://twitter.com/andrewculver/status/1366062684802846721)**
 
-# 🚀 Deployment Process
-The human-essentials & partner application should be deployed ideally on a weekly or bi-weekly schedule. However, this depends on the amount of updates that we have merged into main. Assuming there is updates that we want to ship into deploy, this is the process we take to getting updates from our `main` branch deployed to our servers.
+# Deployment Process 🚀
+The human-essentials & partner application should ideally be deployed on a weekly or bi-weekly schedule depending on the merged updates in the main branch. This is the process we take to deploy updates from our main branch to our servers.
+
+### Requirements
+- SSH access to our servers (usually granted to core maintainers)
+- Login credentials to our [Mailchimp](https://mailchimp.com/) account
+
+
+### Tag & Release
+1. Push a tag with the appropriate semantic versioning. Refer to the [releases](https://github.com/rubyforgood/human-essentials/releases) for the correct versioning. For example, if the last release was `2.1.0` and you're making a hotfix, use `2.1.1`
 
-#### Requirements
-- You will need SSH access to our servers. Access is usually only given to core maintainers of the human-essentials & partner projects.
+    ```sh
+    git tag x.y.z
+    git push --tags
+    ```
+2. Publish a release associated to that tag pushed up in the previous step [here](https://github.com/rubyforgood/human-essentials/releases/new). Include details about the release's updates (we use this to notify our stakeholders on updates via email).
 
-#### Running delayed jobs
+### Running delayed jobs
 
-You can run delayed jobs locally by running the `rake jobs:work` command. You'll need to do this to see any e-mails (they should
-pop up in your browser). Alternatively, you can run a specific delayed job by opening a Rails console and doing something like:
+Run delayed jobs locally with the `rake jobs:work` command. This is necessary to view any emails in your browser. Alternatively, you can run a specific delayed job by opening a Rails console and doing something like:
 
 ```ruby
 Delayed::Job.last.invoke_job
 ```
 
 You can replace the `last` query with any other query (e.g. `Delayed::Job.find(123)`).
 
-#### In-flight Pull Requests
+### Send Update Email To Human Essential Users
+To notify stakeholders about the deployment and updates via email, follow these steps:
 
-Sometimes we want to get a PR up there and going so that other people can review it or provide feedback, but maybe it's incomplete. This is OK, but if you do it, please tag your PR with `in-progress` label so that we know not to review / merge it.
+1. Fetch all the users' emails from our human essentials production database
+```ruby
+cap production rails:console
```

Comment:
This is no longer true, but again I can update this in a separate PR.

Reply:
I added this to the [issue](https://github.com/rubyforgood/human-essentials/issues/3755#issue-1812212931)